airflow/providers/common/sql/operators/sql.py

```diff
@@ -837,7 +837,7 @@ def execute(self, context: Context):
         pass_value_conv = _convert_to_float_if_possible(self.pass_value)
         is_numeric_value_check = isinstance(pass_value_conv, float)
 
-        tolerance_pct_str = str(self.tol * 100) + "%" if self.tol is not None else None
+        tolerance_pct_str = f"{self.tol:.1%}" if self.tol is not None else None
```

Comment:
I think this (and the other change below) can actually be combined into `error_msg` directly